app/code/Magento/Sales/Model/Order.php

```diff
@@ -1816,6 +1816,30 @@ public function getTotalDue()
         $total = $this->priceCurrency->round($total);
         return max($total, 0);
     }
+    
+    /**
+     * Retrieve order total due or cancel label
+     *
+     * @return float|null
+     */
+    public function getTotalDueCancelLabel()
```

Comment:
I do not think that `Order` should be directly responsible for the text of the label.
Could we try to keep this logic isolated in totals block (app/code/Magento/Sales/Block/Adminhtml/Order/Totals.php)?

You can use as an example: https://github.com/nit-it/magento2/pull/2/files

Reply:
@swnsma Please help to verify further.